webapp/package-lock.json

```diff
@@ -6047,9 +6047,9 @@
       },
       "dependencies": {
         "node-fetch": {
-          "version": "2.6.12",
-          "resolved": "https://registry.npmjs.org/node-fetch/-/node-fetch-2.6.12.tgz",
-          "integrity": "sha512-C/fGU2E8ToujUivIO0H+tpQ6HWo4eEmchoPIoXtxCrVghxdKq+QOHqEZW7tuP3KlV3bC8FRMO5nMCC7Zm1VP6g==",
+          "version": "2.7.0",
```

Comment:
This change as well could be in its own PR, i feel.

Reply:
That shouldn't be there, will remove from the commit it got added by mistake